frontend/src/global_styles/content/modules/_avatars.sass

```
@@ -28,4 +28,12 @@
 
 .user-avatar--multi-line
   display: block
-  margin-bottom: 2px
\ No newline at end of file
+  margin-bottom: 2px
+
+.avatar
+  &_placeholder_user
```

```suggestion
  &_placeholder-user
```

An underscore denotes a modifier, of which you can only have one per class

---

I was simply reusing the existing principal types that use dashes. Not too hard to just replace() them though. Will do